doc/man3/OSSL_CMP_validate_msg.pod

```diff
@@ -0,0 +1,98 @@
+=pod
+
+=head1 NAME
+
+OSSL_CMP_cmp_timeframe,
+OSSL_CMP_validate_msg,
+OSSL_CMP_validate_cert_path,
+OSSL_CMP_print_cert_verify_cb
+- functions for verifying CMP message protection
+
+=head1 SYNOPSIS
+
+ #include <openssl/cmp.h>
+ int OSSL_CMP_cmp_timeframe(const ASN1_TIME *start, const ASN1_TIME *end,
+                            X509_VERIFY_PARAM *vpm);
+ int OSSL_CMP_validate_msg(OSSL_CMP_CTX *ctx, OSSL_CMP_MSG *msg);
+ int OSSL_CMP_validate_cert_path(const OSSL_CMP_CTX *ctx,
+                                 X509_STORE *trusted_store, X509 *cert);
+ int OSSL_CMP_print_cert_verify_cb(int ok, X509_STORE_CTX *ctx);
+
+=head1 DESCRIPTION
+
+This is the API for validating the protection of CMP messages,
+which includes validating CMP server certificates and their paths
+while optionally checking the revocation status of the certificates(s).
+
+OSSL_CMP_cmp_timeframe() compares the given time period with the current time.
+If verification parameters are given and X509_V_FLAG_USE_CHECK_TIME is set
+the reference time included there is used instead of the current time.
+
+OSSL_CMP_validate_msg() validates the protection of the given PKIMessage
```

Comment:
PKIMessage or OSSL_CMP_MSG? Similar comment for other occurrences on this page

Reply:
I changed it to C<msg>.